lib/services/donation_service.dart

```diff
@@ -10,12 +12,21 @@ class DonationService {
 
     final String donationId = docRef.id;
     await docRef.update({'id': donationId});
+
+    NotificationModel notification = NotificationModel(
+      title: 'New Donation',
+      body:
+          'A new donation has been made to your campaign: ${donation.amount}\$',
+      userId: donation.receivingOrganizationId,
+    );
+
+    await NotificationService().createNotification(notification);
```

Comment:
let's send this donation notification back to the donor of the donation as well.

Reply:
Done!